lib/screens/home/search/service/service_search_page.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:pet_app/model/service.dart';
+import 'package:pet_app/model/pet.dart';
+import 'package:pet_app/screens/home/search/service/service_search_results.dart';
+import 'package:pet_app/services/auth/auth_service.dart';
+import 'package:pet_app/services/pets/pets_service.dart';
+import 'package:pet_app/services/services.dart';
+
+class ServiceSearchPage extends StatefulWidget {
+  @override
+  _ServiceSearchPageState createState() => _ServiceSearchPageState();
+}
+
+class _ServiceSearchPageState extends State<ServiceSearchPage> {
+  final PetsService _petsService = services.get<PetsService>();
+  final AuthService _authService = services.get<AuthService>();
+  Pet _selectedPet;
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: _petsService
+          .getPetSittablePetsForOwnerId(_authService.currentUserUid),
```

Comment:
Forgot to delete this, the search works though.

Reply:
For real though, it works somehow even with that thing over there, I'll remove it as it's garbage, probably from the category "don't know why it works but it does".

Reply:
Actually I got it, it works because I just need to show the current users' pets in the list so if all of them are "petsittable" it works.